Examples/S3EventNotifier/Sources/main.swift

```diff
@@ -0,0 +1,86 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import AWSLambdaEvents
+import AWSLambdaRuntime
+import AsyncHTTPClient
+
+#if canImport(FoundationEssentials)
+import FoundationEssentials
+#else
+import Foundation
+#endif
+
+let httpClient = HTTPClient.shared
+
+enum LambdaError: Error {
+    case noNotificationRecord
+    case missingEnvVar(name: String)
+
+    var description: String {
+        switch self {
+        case .noNotificationRecord:
+            "No notification record in S3 event"
+        case .missingEnvVar(let name):
+            "Missing env var named \(name)"
+        }
+    }
+}
+
+let runtime = LambdaRuntime { (event: S3Event, context: LambdaContext) async throws -> APIGatewayV2Response in
```

Comment:
S3 Notifications are fire and forget (asynchronous Lambda invocations). We don't need to return a value.  `APIGatewayV2Response` should be `Void`